requests_kerberos/kerberos_.py

```diff
@@ -86,70 +84,75 @@ class HTTPKerberosAuth(AuthBase):
     object."""
     def __init__(
             self, mutual_authentication=REQUIRED,
-            service="HTTP", delegate=False):
+            service="HTTP", delegate=False, force_preemptive=False):
```

Comment:
@rbcarson I suppose it depends on how RFC-pedantic you want to be- from [7235](https://tools.ietf.org/html/rfc7235#section-3.1): 
```The "Authorization" header field allows a user agent to authenticate itself with an origin server -- usually, but not necessarily, after receiving a 401 (Unauthorized) response.``` 
I'm not sure what other use cases people use this auth package for, but I know in my case, preemptive auth is a huge perf win.